cool-core/src/main/java/com/nus/cool/core/util/writer/NativeDataWriter.java

```diff
@@ -57,8 +57,7 @@ public class NativeDataWriter implements DataWriter {
 
     private DataOutputStream out = null;
 
```

Comment:
? should keep each line of code roughly < 80 characters

Reply:
I keep it for less than 120 chars.